index.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="/css/global.css">
+  <link rel="stylesheet" href="/css/home.css">
+  <link rel="stylesheet" as="style" crossorigin href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/static/pretendard.min.css" />
+  <title>판다마켓</title>
+</head>
+<body>
+  <header>
+    <div class="header-container max-container ">
+      <a href="/">
+        <img src="/images/logo.png" alt="판다 로고" />
+      </a>
+      <button class="btn-login" onclick="location.href='/login.html'">로그인</button>
+    </div>
+  </header>
+  <main>
+    <section class="hero">
+      <div class="hero-container">
+        <div class="hero-content">
+          <h2 class="hero-content__title">일상의 모든 물건을<br>거래해 보세요</h2>
+          <button class="btn-items" onclick="location.href='/items.html'">구경하러 가기</button>
+        </div>
+        <img src="/images/hero.png" alt="판다 이미지" class="hero__image" />
+      </div>
+    </section>
+    <section class="hot-item">
+      <div class="hot-item-container">
+        <img src="/images/Img_home_01.png" alt="Img_home_01.png" class="hot-item__image" />
```

Comment:
P3:
이미지의 알트 속성은 이미지가 불러올 수 없거나 스크린 리더기에서 이미지 대신 읽어주는 용입니다.
가능하다면 이미지를 설명하는 텍스트를 기입해주시고 아니라면 비워두셔도 됩니다.

```suggestion
        <img src="/images/Img_home_01.png" alt="쇼핑하는 판다 둘" class="hot-item__image" />
```